src/components/MissingDocAlert.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import {
+    Alert,
+    Box,
+    Button,
+    CloseButton,
+    Flex,
+    Heading,
+    Link,
```

Comment:
wrap with next/link instead, do the following to resolve name conflicts
```suggestion
    Link as ChakraLink,
```